Alignment/TrackerAlignment/plugins/BuildFile.xml

```diff
@@ -35,3 +35,23 @@
   <use   name="root"/>
   <flags EDM_PLUGIN="1"/>
 </library>
+
+<library file="CosmicRateAnalyzer.cc" name="AlignmentCosmicRateAnalyzerPlugins">
+  <use   name="Alignment/TrackerAlignment"/>
```

Comment:
@skeshri this is not needed, I think. From the cc file I cannot see that you use something from this package.